sql/core/src/main/java/org/apache/spark/sql/streaming/Trigger.java

```diff
@@ -50,7 +50,7 @@ public static Trigger ProcessingTime(long intervalMs) {
    *
    * {{{
    *    import java.util.concurrent.TimeUnit
-   *    df.writeStream.trigger(ProcessingTime.create(10, TimeUnit.SECONDS))
+   *    df.writeStream().trigger(Trigger.ProcessingTime(10, TimeUnit.SECONDS))
```

Comment:
Since this is a java-friendly example, we had better use `writeStream()`. And, `ProcessingTime.create` shows deprecation warning.